smoketest/tests/register_token.rs

```diff
@@ -1,28 +1,29 @@
 use codec::Encode;
 use ethers::{
-	core::types::Address,
-	middleware::SignerMiddleware,
-	providers::{Http, Provider},
-	signers::{LocalWallet, Signer},
-	utils::parse_units,
-	utils::rlp::Encodable,
+    core::types::Address,
+    middleware::SignerMiddleware,
+    providers::{Http, Provider},
+    signers::{LocalWallet, Signer},
+    utils::parse_units,
+    utils::rlp::Encodable,
```

Comment:
Let's remove the conversion from tabs to spaces and stick with tabs for consistency.

Reply:
Seems it's wrongly formatted by my IDE, to resolve it move `rustfmt.toml` to top level sharing the same rules.